src/lib/albums/AlbumMenu.svelte

```diff
@@ -149,27 +157,33 @@
                 text="Re-import album"
                 isLoading={isReimporting}
             />
-            <MenuDivider />
-            <MenuOption text="⚡️ Enrich" isDisabled />
-            <MenuOption
-                onClick={fetchArtwork}
-                isLoading={isFetchingArtwork}
-                text="Fetch artwork"
-                description={isFetchingArtwork
-                    ? "Fetching from Wikipedia..."
-                    : "Save to folder as cover.jpg"}
-            />
-            {#if artworkResult}
+            {#if !$rightClickedAlbum.artwork}
```

Comment:
Can you remove this condition? I have used this feature a few times when I wanted to replace the existing artwork, or I manually added a cover.jpg into the folder from elsewhere and wanted to re-scan / re-cache.